src/endpoint-microservice/core-api/core-api-error.ts

```
@@ -0,0 +1,107 @@
+  [HttpStatus.FORBIDDEN]: 'FORBIDDEN',
+  [HttpStatus.NOT_FOUND]: 'NOT_FOUND',
+  [HttpStatus.BAD_REQUEST]: 'BAD_REQUEST',
+};
+
+export function resolveStatusCode(
+  error: unknown,
+  upstreamStatus?: number,
+): number {
+  return (
+    extractErrorStatusCode(error) ?? coerceUpstreamStatus(upstreamStatus)
+  );
```

ESLint

[error] Replace `(⏎····extractErrorStatusCode(error)·??·coerceUpstreamStatus(upstreamStatus)⏎··` with `extractErrorStatusCode(error)·??·coerceUpstreamStatus(upstreamStatus`

(prettier/prettier)